backend/benefit/applications/tests/test_applications_api.py

```diff
@@ -352,10 +362,14 @@ def test_application_post_invalid_data(api_client, application):
     data["status"] = "foo"  # invalid value
     data["bases"] = ["something_completely_different"]  # invalid value
     data["applicant_language"] = None  # non-null required
+    data["company_bank_account_number"] = "FI91 4008 0282 0002 02"  # invalid number
 
     data[
         "company_contact_person_phone_number"
     ] = "+359505658789"  # Invalid country code
+
+    # with translation.override(language):
```

Comment:
Was this commented out code left here by accident?

Reply:
Good catch, fixed. The language is determined by accept-language header during request processing so activating translation before making the request didn't work.